qa/rpc-tests/sc_csw_nullifier.py

```diff
@@ -491,35 +491,38 @@ def run_test(self):
         sc_bal = self.nodes[0].getscinfo(scid2, False, False)['items'][0]['balance']
         assert_equal(sc_bal, sc_cr_amount)
 
+        # TODO: this test has been temporarily disabled because the final behavior will be different
+        # (it will be possible to do a CSW even though there is not an active certificate data hash).
```

Comment:
This check has been disabled because it currently fails. Before the last changes it was working by mistake: at line 463 the new sidechain gets created with a parameter `cswVk2` that is in the wrong position (that is the position of the `mbtrVk`).

After removing the usage of MBTR verification key (and updating the `sc_create()` RPC command) the error popped out and made the test fail.